### ISSUE TITLE
[Bug] Compressor Reactivity button not working in GUI

### ISSUE THREAD

**Issue by p-chan5** [sadko4u/lsp-plugins#332]
Hi Vlad! I was testing the Compresssor plugins series and noticed this bug on all. It seems to be a problem with the interface, that is, this parameter should make the Sidechain source smooth but moving the knob in the GUI or setting a value in the popup box nothing happens.

![001](https://github.com/sadko4u/lsp-plugins/assets/123143444/153e1ec9-ce9d-4c79-9e18-16673d7c53a7)

I tried using Carla and got it to work with Right Click > Edit > Parameters

![002](https://github.com/sadko4u/lsp-plugins/assets/123143444/cf7d5992-1d3c-4ee6-94bb-d25e86a2d1df)

Do you think you can take a look at it? Thank you <3

**Comment by sadko4u** [comment]
Hello! Why do you think that `Reactivity` does not work? `Reactivity` is usless only for `Peak` sidechain mode. On other cases we get the following.

Sidechain graph with `reactivity=0`:
![изображение](https://github.com/sadko4u/lsp-plugins/assets/12834464/33d8ec15-9cce-4442-ad65-bab98f823208)

Sidechain graph with `reactivity-250`:
![изображение](https://github.com/sadko4u/lsp-plugins/assets/12834464/4e7b9401-ea38-4041-90b5-d39552a921b6)

**Comment by sadko4u** [comment]
Fixed this moment for Sidechain, the update will be in 1.2.9 release.

**Comment by sadko4u** [comment]
The plugin interface does the same things for all knobs. If one knob doesn't work properly, then all other knobs should  work improperly because the all parameter update logic is written in one `Knob` controller:

https://github.com/lsp-plugins/lsp-plugin-fw/blob/master/src/main/ctl/simple/Knob.cpp

When user changes the knob, we call `submit_value` method which directly writes the value to the port:
```
https://github.com/lsp-plugins/lsp-plugin-fw/blob/57a6ff220793207f8b7130fa155550e2e8c12b0f/src/main/ctl/simple/Knob.cpp#L253-L257
```

For LV2 implementation, the `set_value` is the following:

https://github.com/lsp-plugins/lsp-plugin-fw/blob/57a6ff220793207f8b7130fa155550e2e8c12b0f/include/lsp-plug.in/plug-fw/wrap/lv2/ui_ports.h#L158-L172

That means, that we call LV2 write function to modify the value for the backend. If new value is not delivered to the backend, then we can conclude that LV2 write function suddenly becomes broken.

I tested the new build in Ardour and did not notice any problems after yesterday patch with minimum/maximum values set to Sidechain.

**Comment by p-chan5** [comment]
I understand, so it was a problem with these values. Thanks for taking the time to fix it. You are still the best without a doubt <3

**Comment by sadko4u** [comment]
Fixed in 1.2.10.
